helm/cas-registration/templates/NetworkPolicy/backendPostgresIngress.yaml

```diff
@@ -11,9 +11,9 @@ spec:
   - from:
     - podSelector:
         matchLabels:
-          release: {{ .Release.Name }}
+          release: {{ include "cas-registration.fullname" . }}
```

Comment:
I think this probably needs to stay as .Release.Name since the matcher is "release", otherwise it may not match properly